app/uk/gov/hmrc/residencenilratebandcalculator/views/rnrbHelpers/inputRadioGroup.scala.html

```diff
@@ -0,0 +1,76 @@
+@*
```

Comment:
How come you moved from the govuk template to a custom radio group?

Reply:
It's not in bootstrap anymore - it's the exact same one that it used to be

Reply:
It was renamed. You can get it from: `@import uk.gov.hmrc.play.views.html.helpers.input_radio_group`